docs/ldml/tr35-keyboards.md

```diff
@@ -556,13 +646,10 @@ An element used to keep track of layout-specific settings by implementations. Th
 
 _Attribute:_ `normalization="disabled"`
 
-> Normalization will not typically be the responsibility of the keyboard author, rather this will be managed by the implementation.
-> The implementation will apply normalization as appropriate when matching transform rules and `<display>` value matching.
-> Output from the keyboard, following application of all transform rules, will be normalized to implementation or application-requested form.
+> The presence of this attrinbute indicates that normalization will not be applied to input text, matching, or output.
+> See [Normalization](#normalization) for additional details.
 >
-> However, it is recognized that there may be some keyboards which, for compatibility or legacy reasons, need to manage their own normalization. The implementation in that case will do no normalization at all. The keyboard author must make use of transforms in the keyboard to any required normalization. In this case, the attribute value `normalization="disabled"` is used to indicate that no automatic normalization happens.
->
-> **Note**: while this attribute is allowed by the specification, its use is discouraged, and keyboards with `normalization="disabled"` would not be accepted into the ClDR repository.
+> **Note**: while this attribute is allowed by the specification, it should be used with caution, and keyboards with `normalization="disabled"` would not be accepted into the ClDR repository.
```

Comment:
```suggestion
> **Note**: while this attribute is allowed by the specification, it should be used with caution, and keyboards with `normalization="disabled"` would not be accepted into the CLDR repository.
```

Though the idea of removing this bit for now, until a proper discussion happens later, was floated in today's meeting.